libs/design/scss/daff-util.scss

```diff
@@ -11,21 +11,18 @@
 //
 // @usage
 // ```
-// @import "daff-util";
+// @import '@daffodil/design/scss/daff-util';
 // ```
 
-// Variables
-@import 'typography/variables';
-@import 'responsive/variables';
+@forward 'core';
+@forward 'typography';
+@forward 'layout';
+@forward 'interactions';
+@forward 'accessibility';
 
-// Daffodil SASS Core
-@import 'core/core';
-
-// Sass Mixins & Functions
-@import 'interaction/mouse/clickable';
-@import 'responsive/breakpoint';
-@import 'typography/mixins';
-@import 'typography/sizes/mixins';
-
-// Accessibility
-@import 'accessibility/errors';
+// @deprecated `@import` is deprecated and will be removed in v1.0.0
+@import 'core';
```

Comment:
Should these be `@import`?

Reply:
See note above: `// @deprecated `@import` is deprecated and will be removed in v1.0.0`

Reply:
removed